src/styles/colors.ts

```diff
@@ -25,7 +26,8 @@ const basicDarkColors: typeof basicLightColors = {
   black: '#ffffff',
   white: '#000000',
   peach: '#FF5B00',
-  liteOrange: '#242424'
+  liteOrange: '#242424',
+  orangeTransparent: '#F6D5C3'
```

Comment:
reuse real alpha color (like `basicLightAlphaColors`)  
and let designer fix this